extensions/src/main/scala/ai/h2o/sparkling/extensions/serde/Utils.scala

```diff
@@ -0,0 +1,36 @@
+/*
+* Licensed to the Apache Software Foundation (ASF) under one or more
+* contributor license agreements.  See the NOTICE file distributed with
+* this work for additional information regarding copyright ownership.
+* The ASF licenses this file to You under the Apache License, Version 2.0
+* (the "License"); you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+package ai.h2o.sparkling.extensions.serde
+
+import water.fvec.Vec
+
+object Utils extends ChunkSerdeConstants {
```

Comment:
note: `Utils` sounds a bit generic. I'm wondering whether we shouldn't go for anything more specific to serialization/deserialization although the class is in the `ai.h2o.sparkling.extensions.serde` namespace. 

This note is  really really minor and you can leave it as it's.

Reply:
Good point, what about `SerdeUtils`?

Reply:
Sounds good to me ;-)